db/db.go

```diff
@@ -199,36 +205,26 @@ func (d DB) UpdatePost(p Post) error {
 	return nil
 }
 
-func sanQry(par map[string]string) string {
-	sort := "date"
-	dir := "asc"
+func sanQry(p Params) string {
```

Comment:
This should be

```go
func (p Params) String() string {
  // ...
}
```
This is the preferred way of converting any type to a string. E.g. https://github.com/nobe4/gh-not/blob/8568098ca9fc97ecc8f2756743259576fb02b6a2/internal/notifications/view.go#L36-L42